lib/helper.js

```diff
@@ -44,7 +45,7 @@ var validationType = function(value, validation, key){
       return {value: value};
     }else{
       if(types[validation].type) validation = types[validation].type;
-      return {error: key + ': ' + value + ', has to be ' + validation + ' type.', param: key};
+      return {error: key + ': ' + escapeHtml(value) + ', has to be ' + validation + ' type.', param: key};
```

Comment:
Express is converting the response itself to `text/html`. GIven that the only possible place for a code injection in this module is through the parsed value, I don't see why the escaping shouldn't happen when its contents are pushed to the error list.